docs/intro.rst

```diff
@@ -134,32 +119,78 @@ by importing the package into Python::
+
+* Approach with ``pywbem_os_setup.sh`` script:
+
+  This is the recommended approach, as of pywbem 0.11.
+
+  In this approach, a standalone script that installs the OS-level
+  prerequisites is downloaded and executed.
+
+  * Download the :download:`pywbem_os_setup.sh </pywbem_os_setup.sh>` script
+    (for pywbem version |version|).
+
+  * Execute that script:
+
+    .. code-block:: bash
+
+        $ ./pywbem_os_setup.sh
+
+* Approach with ``install_os`` command of ``setup.py``:
+
+  This approach was introduced in pywbem 0.8 and has been deprecated with
+  pywbem 0.11. The examples are shown for pywbem version 0.8.1.
+
+  * Download the pywbem source archive from Pypi using Pip:
+
+    .. code-block:: bash
+
+        $ pip download --no-deps --no-binary :all: pywbem==0.8.1
+
+    The ability to download packages was introduced in Pip 8.0. When using an
+    older Pip version, you can download the pywbem source archive from its Git
+    repository:
+
+    .. code-block:: bash
+
+        $ curl -o pywbem-0.8.1.tar.gz -L https://github.com/pywbem/pywbem/blob/master/dist/pywbem-0.8/pywbem-0.8.1.tar.gz
+
+    Of course, you can also download it manually from Pypi. Note that only the
+    source archive (``*.tar.gz``) has the capability we use here; the binary
+    archive (``*.whl``) does not have that capability.
+
+  * Extract the pywbem source archive and run the ``install_os`` command of
+    ``setup.py``:
 
-    $ python setup.py install_os
+    .. code-block:: bash
 
-For a number of well-known Linux distributions, the setup script will install
-the packages using the respective installer program (e.g. ``yum`` on RHEL). For
-other Linux distributions and for non-Linux operating systems, the setup script
-will display the package names that would be needed on RHEL, leaving it to the
-user to translate them accordingly.
+        $ tar -xzf pywbem-0.8.1.tar.gz
+        $ cd pywbem-0.8.1
+        $ python setup.py install_os
 
-When installing such packages on Linux, the setup script uses the ``sudo``
-command, so your userid needs to be authorized accordingly.
+Both approaches support a number of well-known Linux distributions. For these
+supported environments, the OS level packages will be installed using the
+respective installer program for the environment (e.g. ``yum`` on RHEL). For
+other Linux distributions and for non-Linux operating systems, the package
+names for RHEL will be displayed, leaving it to the user to translate them
+accordingly.
 
-This command also downloads and builds ``swig`` and installs its build
-prerequisites if it cannot be installed as an operating system package in the
-required version.
+Both approaches use the ``sudo`` command under the covers, so your userid needs
+to be authorized accordingly, but you don't need to run the script using
+``sudo``.
 
-The second example in the previous section shows how to use the new setup
-command ``install_os``. It is really simple to be used and avoids the manual
```

Comment:
Drop work really. Think the phrase should be "It is simple to use and avoids"

Reply:
IGNORED - this is a comment on removed text.